src/main/webapp/css/main.css

```diff
@@ -45,6 +45,17 @@ nav a {
   width: 800px;
 }
 
+.Avatar {
```

Comment:
Let's stay consistent and user lowercase here

.Avatar --> .avatar

Same for below.